control-plane/subcommand/server-acl-init/command.go

```diff
@@ -668,6 +672,21 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if c.flagDNSProxy {
+		serviceAccountName := c.withPrefix("dns-proxy")
+
+		dnsProxyRules, err := c.dnsProxyRules()
+		if err != nil {
+			c.log.Error("Error templating inject rules", "err", err)
```

Comment:
```suggestion
			c.log.Error("Error templating dns-proxy rules", "err", err)
```
could use const here as well